.github/workflows/issues.yml

```diff
@@ -4,22 +4,76 @@ permissions:
 
 on:
   issues:
-    types: [opened, labeled]
+    types: [opened]
 
 jobs:
-  triage-labels:
+  assignIssue:
+    name: Assign Issue to Someone
     runs-on: ubuntu-latest
+    if: github.repository == 'microsoft/PTVS' && github.event.action == 'opened'
     steps:
-      - name: If no labels, add triage
-        id: no-labels
-        uses: andymckay/labeler@master
-        if: ${{ join(github.event.issue.labels.*.name, ',') == '' }}
+      - name: Find last assigned
+        id: assigned
+        uses: actions/github-script@v6.3.3
         with:
-          add-labels: 'needs triage'
-          ignore-if-assigned: true
-
-      - name: If labeled, remove triage
-        uses: andymckay/labeler@master
-        if: ${{ steps.no-labels.outcome == 'skipped' && join(github.event.issue.labels.*.name, ',') != 'needs triage' }}
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          result-encoding: string
+          script: |
+            const issue = await github.rest.issues.get({
+              issue_number: 7774,
```

Comment:
is this issue number supposed to be hardcoded here (and in other places below)?

Reply:
Yes that hardcoded issue is used to store the last person assigned an issue. Whomever is assigned this issue is the last person to get an incoming issue.